src/main/java/com/amazonaws/encryptionsdk/kmsv2/KmsMasterKeyProvider.java

```diff
@@ -3,33 +3,25 @@
 
 package com.amazonaws.encryptionsdk.kmsv2;
```

Comment:
We can bikeshed the naming some more, but I would prefer `kmssdkv2` to `kmsv2`, as that makes it clearer that the sdk is what is being versioned, not kms itself.

Reply:
Yea, I'd like a better name, I'm not sure 3 abbreviations in a row would be very clear though either. I'll ruminate on some ideas that are clearer.